API/NoDocs/Microsoft.Extensions.Diagnostics.ResourceMonitoring/LinuxResourceUtilizationProviderOptions.cs

```diff
@@ -0,0 +1,16 @@
+// Assembly 'Microsoft.Extensions.Diagnostics.ResourceMonitoring'
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Shared.Data.Validation;
+
+namespace Microsoft.Extensions.Diagnostics.ResourceMonitoring;
+
+public class LinuxResourceUtilizationProviderOptions
+{
+    [TimeSpan(100, 900000)]
+    public TimeSpan CpuConsumptionRefreshInterval { get; set; }
+    [TimeSpan(100, 900000)]
```

Comment:
Our general philosophy is to allow only reasonable values in order to avoid silly configuration errors. Bad configuration is the leading cause of outages, so anything we can do to stop stupid mistakes from screwing things up is a good thing.